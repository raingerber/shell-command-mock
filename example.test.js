
/* eslint-env jest */

describe('example.js', () => {
  it('should resolve without errors', async () => {
    jest.resetModules()
    await expect(require('./example.js')).resolves.toMatchSnapshot()
  })
})
