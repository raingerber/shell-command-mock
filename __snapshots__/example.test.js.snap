// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example.js should resolve without errors 1`] = `
Object {
  "code": 0,
  "signal": "",
  "stderr": "",
  "stdout": "Frog
",
}
`;
